documentation/solution-architecture.md

```diff
@@ -0,0 +1,151 @@
+    - the result of a NLX call stored in the channel of and thus signed with the NLX certificate of the issuer (every claim specifically allowed to specific other actors)
+7) validation channel of validator, destroyed after validation or service shutdown whatever comes first
+      - the same claim and attestation pair as was issued from the delivery channel of the issuer, read from the QR code on a piece of paper
+
+besides these channels, information is issued and stored in supported verifiable credential platforms (in privately held wallets of the client as holder). For now only a custom paper wallet platform is used which stores an attested claim in a QR code printed on a piece of paper to be owned by the client. This piece of paper shall also contain the raw information contained in the attested claim in the QR code which limits the amount of data that can be held in claims such you can issue them in a paper wallet.
+
+The solution just presents, signs and issues information as returned as result from the NLX call; for now it does not semantically parses this information. For now, the webservice called through NLX needs to accept a BSN number as argument that identifies the client though these kinds of arguments are configurable. The object of what is needed or attended to is something configured by the issuer like 'Uittreksel-GBA-Haarlem'. All other data results from usage of the abundance service api and discipl core api and the connectors used (ephemeral and NLX for now).
+
+The process for issuing proofs as described above is illustrated in verifiable claim sequence diagrams below:
+
+![](images/vcseq1.png)
+
+![](images/vcseq2.png)
+
+
+![](images/vcseq3.png)
+
+![](images/vcseq4.png)
+
+## Application layer
+
+The solution is mainly hosted on a central Discipl ephemeral server application hosted by the issuer to which
+multiple client applications (for clients, issuer and administrator) connect to (through the client side Discipl core api).
+
+The issuer runs a waardepapieren service on this server that sets up the ephemeral server and connects to this to attend to needs expressed by other clients on this platform. The following other client applications connect to this ephemeral service hosted by the issuer:
+
+- clerk-frontend
+- authorisation-service
+- client-frontend
+- adminstrator-frontend
+
+Validation does not require usage of this server; validators run their own embedded local ephemeral service (without client-server communication) just to hold information temporarily and use it for their own validation process executed within a client side validation app. As NLX certificates in the end are to originate from official root certificates (conform EiDAS/EIF), validation for now just involves standard techniques around X.509 certificates and can even be done offline.
+
+Below is a more detailed component view:
+
+![](images/components.png)
```

Comment:
It's probably more clear to move the NLX-connector within the wp-service, and then show it connects to the NLX-outway.

At the moment, the ephemeral server is embedded in the wp-service. We might want to make an issue to move it out.

Reply:
yes